src/pages/progress_reports/SubmitProgressReport.tsx

```diff
@@ -136,18 +130,24 @@ export default function SubmitProgressReport(props: SubmitProgressReportProps) {
     const serverError = consentError.message ?? 'Unknown error';
     Notifications.showError({text: message + serverError});
   }
+  const disabledStyle = {
+    backgroundColor: Theme.palette.disabled,
+    borderColor: Theme.palette.disabled
+  }
 
   return (
       <div className='flex flex-row' style={{justifyContent: 'flex-start'}}>
-        <button
-            type={'button'}
-            className='button button-blue'
-            style={{marginRight: '2rem', cursor: 'pointer'}}
-            data-cy='pr-submit-button'
-            onClick={submit}>Submit
-        </button>
-        <button
-            type={'button'}
+        <span>
+            <button type={'button'}
+                className='button button-blue'
+                style={{marginRight: '2rem', cursor: 'pointer', ...(disabled ? disabledStyle : {})}}
+                data-cy='pr-submit-button'
+                disabled={disabled}
+                title='Complete required form fields to enable submission.'
+                onClick={submit}>Submit
+            </button>
```

Comment:
```suggestion
                onClick={submit}>
                Submit
            </button>
```

Improve readability.